### PR TITLE
Feat/styling

### PR DESCRIPTION
**Pull request by Lino09**
I this PR I used some nice colors to give the page a nice dark look
I also added some neon buttons which I think looks gorgeous

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -39,25 +36,25 @@ <h2 class="subtitle">Recent scores (Quidditch Game)</h2>
             </div>
             <div>
                 <h3 class="subtitle">Add your score</h3>
-                <form action="POST" class="add-score-form">
+                <form action="POST" class="add-score-form" autocomplete="off">
                     <label class="add-score-label" for="name">
-                        <input class="add-score-input" type="text" id="name" name="Your name" required
+                        <input class="add-score-input" type="text" id="name" name="Your name" required placeholder="Your name"
```
Comment (rloterh) [inline src/index.html]: - [x] OPTIONAL
The _Your name_ input field is able to accept numbers whiles _Your score_ input field accepts letters. Kindly endeavor to provide input validations for the same.

File: README.md [hunk 2]
```diff
@@ -2,21 +2,19 @@
...
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS & JavaScript
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://lino09.github.io/leaderboard/dist)
```
Comment (rloterh) [inline README.md:18]: - [x] OPTIONAL
Kindly change the  heading title `Authors` to `Author` [ on line 40 of the ReadMe], so as to make your ReadMe professional

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Abel,  

Your project is complete and it's fantastic! 👏  Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- App is running as intended
- Good ReadMe
- Nice UI design
- No linter errors

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review:_
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Lino09) [reply README.md:18]: Thanks for your help! I made the changes

**Pull request merged**